Resources/Prototypes/Entities/Structures/Power/apc.yml

```
@@ -1,5 +1,5 @@
 - type: entity
-  abstract: true
+  noSpawn: true
```

```suggestion
  abstract: true
```
Should not be used otherwise.